README.md

```diff
@@ -10,13 +10,15 @@ See [this page](https://docs.docker.com/engine/getstarted/step_one/) for more in
 
 * [Docker for Mac](https://download.docker.com/mac/stable/Docker.dmg), or
 * [Docker for Windows](https://download.docker.com/win/stable/InstallDocker.msi), or
-* [Docker for Linux](https://docs.docker.com/engine/installation/)
+* [Docker for Linux](https://docs.docker.com/engine/installation/), or
+* [Docker Toolbox for Mac/Windows](https://www.docker.com/products/docker-toolbox) (a/k/a _docker-machine_)
 
-"Docker for Mac" and "Docker for Windows" place requirements significant requirements on the mimimum required OS version and features.  An additional flavor of Docker (which we'll refer to as _docker-machine_) runs Docker inside Virtualbox, and can be run on almost any OS.  Unfortunately, the evaluation instructions differ slightly for _docker-machine_.   
-* [Docker Toolbox for Mac/Windows](https://www.docker.com/products/docker-toolbox)
-  * You need to start an instance of docker-machine to run the docker containers.  See the [create a machine](https://docs.docker.com/machine/get-started/) documentation for how to do that.
+"Docker for Mac" and "Docker for Windows" place significant requirements on the mimimum required OS version and features.
 
```

Comment:
I have the para break there on purpose.  So that people realize that there is a difference between "Docker for *" and Docker machine.